tasks/config.ts

```diff
@@ -87,18 +85,17 @@ export function load() {
     const blockchainConfig = {
       ["BDX_ADDRESS"]: (await getBdx(hre)).address,
       ["NATIVE_TOKEN_WRAPPER_ADDRESS"]: (await getWeth(hre)).address,
-      ["EXTERNAL_USD_STABLE"]: EXTERNAL_USD_STABLE[hre.network.name],
+      [`EXTERNAL_USD_STABLE`]: EXTERNAL_USD_STABLE[hre.network.name],
```

Comment:
did you change it back from " to ` by purpose?
relevant for all the fields of this object